src/dotnet/Chat.UI.Blazor/Components/RecordingSubHeader/RecordingSubHeader.razor

```diff
@@ -7,26 +7,31 @@
     if (ReferenceEquals(m, Model.NotRecording))
         return;
     var recordingElsewhereCls = m.RecordingChat != null ? "recording-elsewhere" : "";
+    var isWarning = m.RecordingChat == null;
 }
 
 <div class="recording-sub-header @recordingElsewhereCls">
     <div class="c-round-container">
-        <div class="c-round"></div>
+        @if (isWarning) {
+            <i class="icon-alert-triangle text-banner-warning"></i>
+        } else {
+            <div class="c-round"></div>
+        }
     </div>
 
-    <div class="c-line-1">
-        @if (m.RecordingChat != null) {
+    @if (m.RecordingChat != null) {
```

Comment:
Did you check how it looks like when RecordingChat != null and StopRecordingAt != null ?